engine/schema/src/main/resources/META-INF/db/schema-41720to41800.sql

```diff
@@ -1035,3 +1035,10 @@ WHERE   role_id = (SELECT id FROM `cloud`.`roles` WHERE name = 'Read-Only User -
 INSERT INTO `cloud`.`role_permissions` (`uuid`, `role_id`, `rule`, `permission`)
 SELECT UUID(), `roles`.`id`, 'isAccountAllowedToCreateOfferingsWithTags', 'ALLOW'
 FROM `cloud`.`roles` WHERE `role_type` = 'DomainAdmin';
+
+--- Create table for handling allowed console session temporally
+
+CREATE TABLE `cloud`.`allowed_console_session` (
```

Comment:
may be better to use `allowed_vm_console_sessions` ?

Reply:
@weizhouapache, the table was redesigned, vide https://github.com/apache/cloudstack/pull/7094#issuecomment-1385309721.